src/main/java/seedu/address/model/transaction/Transaction.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.model.transaction;
+
+import seedu.address.model.person.Person;
+import seedu.address.model.person.UniquePersonList;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+
+import java.util.Objects;
+import java.util.Set;
+
+public class Transaction {
```

Comment:
This class and its non-trivial methods should have header comments. 
If not, developers who write client code for this class will not know how to use it.